components/frontend/src/dashboard/StatusBarChart.js

```diff
@@ -19,7 +21,7 @@ export function StatusBarChart({ animate, colors, label, tooltip, summary, maxY,
         })
         return (
             <VictoryBar
-                barRatio={barRatio}
+                barWidth={(props) => ratio * (props.width - nrDates * 2 * props.style.padding - 2 * horizontalPadding) / nrDates}
```

Comment:
Does this work the same on retina and regular displays?

Reply:
Unfortanately not. It looked great on my Retina display, but not a wide screen. Will fix this.